app/services/copy_profiles_to_policies.rb

```diff
@@ -22,7 +22,7 @@ def profiles
   def create_policies
     profiles.where(external: false).find_each do |profile|
       policy = Policy.create!(Policy.attrs_from(profile: profile))
-      profile.update!(policy_id: policy.id)
+      profile.update!(policy_id: policy.id, business_objective_id: nil)
```

Comment:
Since we haven't deployed this yet, it makes sense to just unassign the BO from the profile so we don't face issues with dangling foreign keys.

Reply:
Haven't we deployed it to stage already?

Reply:
Yes, but with a prod dump DB just to test the profile to policy migration. It will need to be re-run there against the normal stage DB.